### PR TITLE
feature about section of the event occurrence screen and organization …

### LINKED ISSUE #97: Feat: "About" section of the Event Occurrence Screen and Organization Links
Hey Joji! Happy to have you working on some eng. work!

Before starting this ticket I recommend that you read all relevant documentation (pretty much all of it unfortunately), familiarize yourself with the codebase and ask as many questions as possible. This ticket will span both backend changes and frontend changes so you'll need to have an understanding of the entire stack. If it is your first time using Golang, I recommend taking the tour [here](https://go.dev/tour/welcome/1). If its your first time using expo or working on mobile, I recommend brushing up on some of the docs, in particular the file structure [here](https://docs.expo.dev/router/basics/notation/#_layouttsx-files).

When you are about to get started shoot Ally and me a DM and we can send over the .env.

## Description

The goal of this ticket is to extend the existing event occurrence screen to include an "About" section(see the attached screenshot). The description of an event is sourced from the parent event. We also want organizations to be able to store relevant links and to display these links on the event occurrence. These should be stored somehow within the event table. Make a design decision as to how this data should be stored. We should support users to enter arbitrary strings for the link href and the link label.

Image

### Backend Changes: 
- [x] Make a design decision as to how we will store the links for links on organizations and send the proposed changes to Ally and I
- [x] Create a new migration to extend the organization with the proposed changes
- [x] Update the existing types for the GET/POST API endpoints for Organizations as well as the GET endpoint for an event occurrence
- [x] Update the GET/POST/PATCH endpoints for organizations so that we return/update a list of link, link description pairs to the client
- [x] Update the GET endpoint for event occurrences so that we return a list of link, link description pairs to the client
- [x] Update  the Huma docs for the API (This will ensure that they given payload follows the types from the above step)
- [x] Run 'make api-gen' to update the openapi spec and make sure the updates to the openapi.yaml are correct
- [ ] Manually test endpoint in scalar or something like postman
- [x] Run the linter
- [ ] Add screenshots of manual tests to PR comment

### Frontend Changes:
- [ ] Extend the existing event occurrence screen to display the "about me" section as shown in the figma(and the attached screenshot). 
- [ ] Dynamically render the buttons for each of the links that are present within the event occurrence payload with its associated label
- [ ] When a user presses one of these buttons, we should route the user to their default browser. Take a look at this [link](https://docs.expo.dev/versions/latest/sdk/webbrowser/)
- [ ] Take screenshots of the functionality and include them in a comment on this PR

Ensure that any non-trivial logic is abstracted behind react hooks. Please also be sure to use nativewind rather than stylesheets

### PR DESCRIPTION
**Pull request by Jojister**
…links 97

Closes #97

# Design Decision
Stored org links as a JSONB column on the `organization` table as an array of `{href, label}` objects. Links are always fetched/updated with the org row so no join table is needed.

# Backend Changes
- New migration: `add_links_to_organizations`
- Added `OrgLink` type to models
- Updated GET/POST/PATCH org endpoints to return/accept `links`
- Updated GET event occurrence endpoints to return `org_links` from parent org
- Updated Huma docs and regenerated `openapi.yaml`

# Frontend Changes
- Added `AboutSection` component with `useOrgLinks` hook
- Dynamically renders link pills from `org_links` payload
- Opens links in default browser via `expo-web-browser`
- NativeWind styling, no StyleSheets
- Regenerated API client types via orval

# Screenshots

# Checklist

## General
- [y ] I have performed a self-review of my code
- [y] I have commented my code, particularly in hard-to-understand areas

## Backend Changes (if applicable)
- [y ] I have written unit tests for my code
- [y ] I have written integration tests where appropriate
- [y ] New and existing unit tests pass locally with my changes
- [y ] I have run make api-validate and make api-gen to keep the openAPI spec up to date
- [y ] I have tested error handling and edge cases
- [y ] Database migrations are included (if applicable)

## Frontend Changes (if applicable)
- [n/a ] I have tested in multiple browsers (Chrome, Firefox, Safari)
- [y ] Mobile responsive design verified (if applicable)
- [y ] I have run make generate-api to keep the API hooks up to date
- [y ] **For mobile features:** Tested on both iOS and Android devices
- [y ] **For mobile features:** Screenshots from both iOS and Android included below
- [y ] No console errors or warnings
- [y ] Loading states and error states handled

## Mobile Screenshots (if applicable)
Screenshot 2026-03-29 at 2 04 32 PM
Screenshot 2026-03-23 at 1 13 14 PM
Screenshot 2026-03-23 at 1 13 35 PM
Screenshot 2026-03-23 at 1 13 52 PM
Screenshot 2026-03-25 at 2 06 04 PM

----------------------------------------------
image
image

### CODE REVIEW

File: frontend/apps/mobile/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "skillspark",
   "version": "1.0.0",
   "scripts": {
-    "start": "bash ../../../scripts/load-root-env.sh expo start --tunnel",
+    "start": "bash ../../../scripts/load-root-env.sh expo start",
```
[the remark this answers is not in the document]
Comment (adescoteaux1) [reply frontend/apps/mobile/package.json]: done

**Review by josh-torre — APPROVED** [review]
LGTM 🚀
